config/routes.rb

```diff
@@ -3,4 +3,13 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # config/routes.rb
+
+  resources :amusement_parks do
+    resources :rides
+  end
+
+  resources :mechanics do
+    resources :rides, through: :ride_mechanics
```

Comment:
Nice nesting! THis is the route that would be better to use for the form to add a ride to a mechanic. 

You're also welcome to handroll routes in an IC, so it would be a post to "/mechanics/:mechanic_id/rides"